### PR TITLE
Fix import cycle

### PR DESCRIPTION
**Pull request by sujayakar**
I moved some of the input handler code out of `inputs.ts` just to be safe/symmetric too.

There's also another tiny fix where we set players' velocity to zero when they've stopped moving.

### CODE REVIEW

File: convex/game/agents.ts [hunk 1]
```diff
@@ -10,7 +10,8 @@ import {
 import { Doc } from '../_generated/dataModel';
 import { Players, activity } from './players';
 import { AiTown } from './aiTown';
-import { inputHandler, join } from './inputs';
+import { join } from './inputs';
```
Comment (ianmacartney) [inline convex/game/agents.ts:13]: is this still a cycle?

Comment (ianmacartney) [reply convex/game/agents.ts:13]: I think you want to use `joinGame` from `./players`

File: convex/game/inputs.ts [hunk 2]
```diff
@@ -107,6 +53,8 @@ export const inputs = {
   moveTo,
   // Inputs for the messaging layer.
   ...conversationInputs,
+  startConversation: conversationInputs.startConversation,
+
```
Comment (ianmacartney) [inline convex/game/inputs.ts:57]: is this still necessary?

File: convex/game/inputs.ts [hunk 3]
```diff
@@ -1,21 +1,13 @@
-import { blocked } from './movement';
-import { characters } from '../../data/characters';
 import { agentInputs } from './agents';
-import { ObjectType, PropertyValidators, v } from 'convex/values';
+import { ObjectType, v } from 'convex/values';
 import { movePlayer } from './movement';
-import { stopConversation } from './conversations';
 import { conversationInputs } from './conversations';
 import { point } from '../util/types';
 import { AiTown } from './aiTown';
```
Comment (ianmacartney) [inline convex/game/inputs.ts:6]: we could make this `import type {AiTown} ...` to help out with that cycle too

File: convex/game/agents.ts (hunk 1 above)
Comment (sujayakar) [reply convex/game/agents.ts:13]: whoops, thanks!

File: convex/game/inputs.ts (hunk 2 above)
Comment (sujayakar) [reply convex/game/inputs.ts:57]: removed, thx

**Pull request merged**